src/i_gamepad.h

```diff
@@ -27,6 +27,14 @@
 #define RAD2TIC(x) ((x) * 32768.0f / PI_F) // Radians to ticcmd angle.
 #define LENGTH_F(x, y) sqrtf((x) * (x) + (y) * (y))
 
+typedef enum joy_platform_e
```

Comment:
This is cool! Could we get something similar for the Nintendo Switch and its Y/X/B/A button names? Or does this already match with the XBox layout?

Reply:
Sure, I can add Switch naming. The current differences are minor (e.g. back/start in Woof instead of minus/plus on a Switch controller). You may have noticed that Switch's A/B and X/Y are in different positions than the common standard, but it's still A as confirm and B as cancel. That's just how Nintendo does things.

Reply:
Yes, right. But I guess it would still help if we had presets for the three most common consoles out there.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1800/commits/8552e0ec5ee015895e338378813500306286d58d